app/serializers/api/v1/beneficiary_serializer.rb

```diff
@@ -0,0 +1,16 @@
+
+module Api::V1
+  class BeneficiarySerializer < ApplicationSerializer
+    attributes  :id,
+                :identity_type_id,
+                :identity_number,
+                :created_by_id,
+                :title,
+                :first_name,
+                :last_name,
+                :phone_number
+
+    has_one  :created_by, serializer: UserSerializer, root: :user
+    has_one  :identity_type, serializer: IdentityTypeSerializer, root: :identity_type
```

Comment:
Do we really need `root: :identity_type` here ? I think it will work without it as serializer name is same as what we have written in `root`.

`has_one  :created_by, serializer: UserSerializer, root: :user`

here we require root as we do not have created_by serializer  and we have UserSerializer thus `root: :user` is needed.

But for identity_type that's not case.